src/Server.ts

```diff
@@ -1,21 +1,28 @@
-const { once } = require('events');
-const path = require('path');
-const express = require('express');
-const debug = require('debug');
-const configImport = require('./config.js');
+import { once } from 'events';
+import path from 'path';
+import express from 'express';
+import debug from 'debug';
+import { router } from './routes/router';
 
-const config = configImport.config();
 const info = debug('team-link:info');
 const error = debug('team-link:error');
 
-class Server {
-    constructor(app, port, host, staticPath) {
+export default class Server {
+    app
+    host
+    port
+    staticPath
+    server
+
+    constructor ({ app, port, host, staticPath }) {
         this.app = app;
         this.host = host;
         this.port = port;
-        this.staticPath = path.join(__dirname, staticPath);
+        this.staticPath = path.join(path.resolve(), staticPath);
```

Comment:
You can simplify it with `path.resolve(staticPath)`.

Reply:
That does not work as intended:
path.join(path.resolve(), staticPath) -> D:\Training\Team-Link\resources
path.resolve(staticPath) -> D:\resources

Reply:
path.resolve(staticPath) now works (staticPath should not start with '/')